CONTRIBUTING.md

```diff
@@ -35,9 +35,20 @@ in this document.
  * Types, enums, and constants defined in common Windows header files should be defined
    in the PInvoke.Windows.Core project.
 
-When introducing support for a new native DLL to this project, use the templates\AddNewLibrary.ps1
+When introducing support for a new native DLL to this project, use the `templates\AddNewLibrary.ps1`
 Powershell cmdlet to create the projects necessary to support it and follow the instructions from that script.
-The library should also be added to the list on the [readme](README.md).
+
+Sometimes it might be needed to add libraries to share types and structures among high-level API sets.
+When that happens you should use the `templates\AddNewCoreLibrary.ps1` Powershell cmdlet to create a core library.
+
+A Core Library on PInvoke project wording is a library that only contains types and structures, no functions, classes or methods.
```

Comment:
> contains types and structures

structures _are_ types. What other types did you have in mind? enums? Would it be better to say "enums and structures"?